libmamba/src/api/list.cpp

```diff
@@ -152,6 +153,7 @@ namespace mamba
             { return (regex.empty() || std::regex_search(pkg_info.name, spec_pat)); };
 
             auto all_records = prefix_data.all_pkg_mgr_records();
+            auto requested_specs = prefix_data.history().get_requested_specs_map();
```

Comment:
```suggestion
            const auto requested_specs = prefix_data.history().get_requested_specs_map();
```

I cant find a modification for this so I suspect `const` works here.